Lib/unittest/loader.py

```diff
@@ -84,9 +84,16 @@ def loadTestsFromTestCase(self, testCaseClass):
             raise TypeError("Test cases should not be derived from "
                             "TestSuite. Maybe you meant to derive from "
                             "TestCase?")
-        testCaseNames = self.getTestCaseNames(testCaseClass)
-        if not testCaseNames and hasattr(testCaseClass, 'runTest'):
-            testCaseNames = ['runTest']
+        if (
+            issubclass(testCaseClass, case.TestCase)
+            and testCaseClass in (case.TestCase, case.FunctionTestCase)
+        ):
```

Comment:
`issubclass()` is not needed.
```suggestion
        if testCaseClass in (case.TestCase, case.FunctionTestCase):
```